linux_os/guide/services/ntp/chronyd_run_as_chrony_user/rule.yml

```diff
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8,fedora
+
+title: 'Ensure thatchronyd is running under chrony user account'
+
+description: |-
+    chrony is a daemon which implements the Network Time Protocol (NTP). It is designed to
+    synchronize system clocks across a variety of systems and use a source that is highly
+    accurate. More information on chrony can be found at
+    {{{ weblink(link="http://chrony.tuxfamily.org/") }}}.
+    Chrony can be configured to be a client and/or a server.
+    To ensure that chronyd is running under chrony user account, Add or edit the
+    <tt>OPTIONS</tt> variable in <tt>/etc/sysconfig/chronyd</tt> to include ' -u chrony ':
```

Comment:
Are the spaces around the single quotes expected?
`' -u chrony '

Reply:
no spaces should not be there